web/src/styles/global.css

```
@@ -0,0 +1,521 @@
+  --bg-secondary: #1a1d23;
+  --bg-card: #2a2d35;
+  --bg-hover: #32363e;
+  --text-primary: #ffffff;
+  --text-secondary: #8f98a0;
+  --accent-blue: #66c0f4;
+  --accent-blurple: #5865f2;
+  --accent-green: #a4d007;
+  --accent-metacritic: #6dc849;
+  --accent-red: #ed4245;
+  --border: #2a2d35;
+  --font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)